sgkit/utils.py

```diff
@@ -64,3 +64,55 @@ def encode_array(x: ArrayLike) -> Tuple[ArrayLike, List[Any]]:
     rank = np.empty_like(index)
     rank[index] = np.arange(len(index))
     return rank[inverse], names[index]
+
+
+def split_array_chunks(n: int, blocks: int) -> List[int]:
+    """Compute chunk sizes for an array split into blocks.
+    This is similar to `numpy.split_array` except that it
+    will compute the sizes of the resulting splits rather
+    than explicitly partitioning an array.
+    Parameters
```

Comment:
We should add a docstring linter. We can open a separate issue for this. But here we are missing a new line, also before `Examples`, and `Returns`.

Reply:
That would be awesome, I think those keep getting introduced when I use the VSCode conflict resolution feature.  I'll fix that docstring at least.

Reply:
Issue for it: https://github.com/pystatgen/sgkit/issues/70

Reply:
https://github.com/pystatgen/sgkit/pull/69/commits/bda112f96279b909983e3b9c7e47234d6ea9a973